augur/augur_client.py

```diff
@@ -181,11 +181,9 @@ def filter_blocks(self, start_block, end_block, filter, handler,
     '''
     self._require_is_open()
     if (start_block < 0) or (end_block < 0) or (increment < 0):
```

Comment:
Can ```increment``` be 0?

Reply:
`increment` is used to increase the blocks to search in. Setting it to 0 would not allow `_filter_blocks` to crawl through the blockchain. However, an `increment` of 0 could be used for a small range of blocks between `fromBlock` and `toBlock`, and would not need to enter the for loop.

Reply:
What is the formal definition of a small range for which an increment of 0 works?

Reply:
There is no formal definition for what a small range for the increment to be, since the user can define how fast they want to search through the blockchain. Before finding out that there was a payload size limit on the websocket, I thought the timeout was a result of the range between `fromBlock` and `toBlock` was too big and processing the data was taking too long. To workaround the timeout, I set a base `increment` value of 1000 so that the websocket would not timeout and it would search the blockchain at a reasonable pace (rather than incrementing by 1 block at a time).
Since the timeout is a result of the payload size being larger than the default websocket size limit, `_filter_blocks` should be able to receive an `increment` of 0. A small range then, would be one that would return a payload from the websocket that is less than the set websocket's payload size.

Reply:
Carefully trace through your code to determine what it means for an ```increment``` to be ```0``` and fix the bug.